pkg/collector/corechecks/ebpf/probe/tcp_queue_length_test.go

```diff
@@ -25,14 +25,20 @@ import (
 
 var (
 	missingBTFS = map[string]struct{}{
-		"4.18.0-1018-azure":            {},
-		"4.18.0-147.43.1.el8_1.x86_64": {},
+		"4.14.275-207.503.amzn2.aarch64": {},
+		"4.18.0-1018-azure":              {},
+		"4.18.0-147.43.1.el8_1.x86_64":   {},
 	}
 )
 
-func isMissingBTF(kv string) bool {
+func isMissingBTF(cfg *ebpf.Config, kv string) bool {
 	_, ok := missingBTFS[kv]
-	return ok
+	if ok {
+		return true
+	}
+
+	btfData, _ = GetBTF(cfg.BTFPath, cfg.BPFDir)
```

Comment:
I don't think we want this, because we want to fail loudly if BTF is missing when it is expected.